packages/api-client/src/ems.ts

```diff
@@ -0,0 +1,67 @@
+import { lambda } from '@cumulus/aws-client/services';
+import { invokeApi } from './cumulusApiClient';
+import { InvokeApiFunction as emsApi } from './types';
+
+export type EmsReportType = 'distribution' | 'ingest' | 'metadata';
+
+export interface CreateEmsReportsRequest {
```

Comment:
todo: what's using this

Reply:
https://github.com/nasa/cumulus/blob/0e0ca7e8d94e035f013c68e12caecff857ca1c5f/packages/api-client/src/ems.ts#L50